tdrs-backend/tdpservice/email/email.py

```diff
@@ -14,65 +12,18 @@
 logger = logging.getLogger(__name__)
 
 
```

Comment:
NIP: I am not sure how having email_helper.py is a better solution than just having one email.py. In case we need to add more email notifications, then all the emails will go to the helper file which causes us to end up in the same situation of having a long file. It might a good idea to rename the email_helper file based on type of notification and add associated methods to that file.

Reply:
great suggestion, i agree having a "helpers" module that has one file per helper would be much cleaner. i'll get on that.